apigee_edge_ui.module

```diff
@@ -57,25 +64,34 @@ function apigee_edge_ui_theme(array $existing, string $type, string $theme, stri
 }
 
 /**
- * Prepares variables for developer_app template.
+ * Implements hook_entity_type_alter().
  */
-function template_preprocess_developer_app(array &$variables): void {
-  // Helpful $content variable for the developer_app template.
-  $variables['content'] = [];
-  foreach (Element::children($variables['elements']) as $key) {
-    $variables['content'][$key] = $variables['elements'][$key];
+function apigee_edge_ui_entity_type_alter(array &$entity_types): void {
+  /** @var $entity_types \Drupal\Core\Entity\EntityTypeInterface[] */
+  if (isset($entity_types['developer_app'])) {
+    $entity_types['developer_app']->setListBuilderClass('Drupal\apigee_edge_ui\BetterAppListBuilder');
```

Comment:
import the class and use `BetterAppListBuilder.class` because that way static code analyzers can find this place as a used place of this class (and the others below).